### ISSUE TITLE
increased hepatocyte karyomegaly

### ISSUE THREAD

**Issue by sbello** [obophenotype/mammalian-phenotype-ontology#2244]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
increased hepatocyte karyomegaly

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
an increase in the number or proportion of liver cells with enlarged nuclei
http://medical-dictionary.thefreedictionary.com/karyomegaly
PMID:7731963, PMID:22733568,  PMID:8701981, PMID:26980189

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

MP:0000607

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
